backend/data_tools/data/can_data.json5

```diff
@@ -1,626 +1,1022 @@
 {
+  can_funding_details: [
+    {
+      // 1
+      fiscal_year: 2023,
+      fund_code: "AAXXXX20231DAD",
+      funding_source: "OPRE",
+    },
+    {
+      // 2
+      fiscal_year: 2021,
+      fund_code: "BBXXXX20215DAD",
+    },
+    {
+      // 3
+      fiscal_year: 2022,
+      fund_code: "CCXXXX20225DAD"
+    },
+    {
+      // 4
+      fiscal_year: 2021,
+      fund_code: "DDXXXX20215DAD",
+    },
+    {
+      // 5
+      fiscal_year: 2021,
+      fund_code: "EEXXXX20215DAD",
+    },
+    {
+      // 6
+      fiscal_year: 2021,
+      fund_code: "FFXXXX20215DAD"
+    },
+    {
+      // 7
+      fiscal_year: 2023,
+      fund_code: "GGXXXX20231DAD"
+    },
+    {
+      // 8
+      fiscal_year: 2023,
+      fund_code: "HHXXXX20231DAD"
+    },
+    {
+      // 9
+      fiscal_year: 2023,
+      fund_code: "IIXXXX20231DAD"
+    },
+    {
+      // 10
+      fiscal_year: 2023,
+      fund_code: "JJXXXX20231DAD"
+    },
+    {
+      // 11
+      fiscal_year: 2023,
+      fund_code: "KKXXXX20235DAD"
+    },
+    {
+      // 12
+      fiscal_year: 2022,
+      fund_code: "LLXXXX20225DAD"
+    },
+    {
+      // 13
+      fiscal_year: 2023,
+      fund_code: "MMXXXX20235DAD"
+    },
+    {
+      // 14
+      fiscal_year: 2023,
+      fund_code: "NNXXXX20231DAD"
+    },
+    {
+      // 15
+      fiscal_year: 2023,
+      fund_code: "OOXXXX20235DAD"
+    },
+    {
+      // 16
+      fiscal_year: 2023,
+      fund_code: "PPXXXX20235DAD"
+    },
+    {
+      // 17
+      fiscal_year: 2023,
+      fund_code: "QQXXXX20235DAD"
+    }
+  ],
   can: [
     {
+      // 500
       number: "G99HRF2",
       description: "Healthy Marriages Responsible Fatherhood - OPRE",
-      nickname: "HMRF-OPRE",
-      expiration_date: "9/1/2023",
-      appropriation_date: "10/1/2022",
-      arrangement_type: "OPRE_APPROPRIATION",
-      authorizer_id: 26,
-      managing_portfolio_id: 6,
-      division_id: 5,
-      funding_sources: [
-        {
-          "tablename": "funding_source",
-          "id": 26
-        },
-        {
-          "tablename": "funding_source",
-          "id": 24
-        }
-      ]
-    },
-    {
+      nick_name: "HMRF-OPRE",
+      //      expiration_date: "9/1/2023",
```

Comment:
Is there any reason we're keeping this test data in comments instead of deleting it? Will it be used for changed models in future PR?

Reply:
Good catch - I need to delete the comments in the test data.